src/plugins/workspace/server/saved_objects/workspace_id_consumer_wrapper.ts

```diff
@@ -64,7 +141,14 @@ export class WorkspaceIdConsumerWrapper {
         ),
       delete: wrapperOptions.client.delete,
       find: (options: SavedObjectsFindOptions) =>
-        wrapperOptions.client.find(this.formatWorkspaceIdParams(wrapperOptions.request, options)),
+        wrapperOptions.client.find(
+          this.formatWorkspaceIdParams(
+            wrapperOptions.request,
+            // The `formatFindParams` is a workaroud for 2.14 to always list global data sources,
+            // should remove this workaround in 2.15 once readonly share is available
```

Comment:
Let's say `in the upcoming release` which doesn't sound too arbitrary. And we can add `TODO: ` in front of the comment.

Reply:
Sure, done for that.